.travis.yml

```diff
@@ -1,4 +1,8 @@
 language: java
+jdk:
+  - oraclejdk8
+  - oraclejdk7
+  - openjdk7
```

Comment:
It will be tested 3 times, but  the tests are run in parallel, so there should be minimal impact to completion time.